plugins/user-settings/config.d.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /**
+   * Generic frontend configuration.
+   */
+  app: {
+    /**
+     * A list of application feature flags
+     */
+    featureFlags?: Array<{
+      /**
+       * Feature flag name
+       * @visibility frontend
+       */
+      name: string;
+      /**
+       * Optional feature flag description
+       * @visibility frontend
+       */
+      description?: string;
+    }>;
+  };
+  /**
+   * Configuration that provides information on available configured authentication providers.
+   */
+  auth?: {
```

Comment:
Don't think auth should be in here?

Reply:
It was previously in the package.json under configSchema so moved it here